OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
In that case, put @GregHanson for consul, @ijsnellf for eureka, @ayj for kube, in addition to me and Kuat. I am probably missing more folks but these should be a good coverage for starters and we can add people as time progresses.